Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2.yml

```
@@ -177,6 +177,11 @@ configuration:
   name: feedTags
   required: false
   type: 0
+- additionalinfo: The Default API Root to use (e.g. default, public). When remained empty, the server default API root is used, or the first API root if no default is defined.
```

Please make sure to add this to the readme.md.
```suggestion
- additionalinfo: The Default API Root to use (e.g. default, public). If left empty, the server default API root will be used. When the server has no default root, the first available API root will be used.
```